src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -967,6 +967,30 @@ export function buildJsFormTemplate(dotNetFormTemplate: any): FormTemplate {
     return formTemplate;
 }
 
+export function buildJsTimeSliderStops(dotNetStop: any): any | null {
```

Comment:
I wonder if we should start a `jsBuilderPro.ts`, just to keep unecessary code out of Core?